src/hooks/useSendForm.tsx

```diff
@@ -180,24 +178,27 @@ function fromChainReducer(state: FormState, chainId: ChainId): FormState {
   const config = getConfig();
   let fromChain = chainId;
   // reset toChain
-  let toChain = undefined;
+  let toChain = state.toChain;
   let tokenSymbol = state.tokenSymbol;
 
-  let availableRoutes = config.filterRoutes({
-    toChain,
+  // availble routes are all possible routes with this from chain selected
+  const availableRoutes = config.filterRoutes({ fromChain });
+  // selected routes are all possible chains given the users selection from chain and token symbol.
+  let selectedRoutes = config.filterRoutes({
     fromChain,
+    toChain,
     fromTokenSymbol: tokenSymbol,
   });
-
-  // we have from, to, symbols that produce no routes, so reset the form to the first valid route
-  if (!availableRoutes.length) {
-    availableRoutes = config.filterRoutes({ fromChain });
-    const [firstRoute] = availableRoutes;
-    toChain = firstRoute.toChain;
-    tokenSymbol = firstRoute.fromTokenSymbol;
+  // if no valid routes found with user selection, de-prioritize the toChain to maintain token symbol.
+  if (!selectedRoutes.length) {
```

Comment:
nit: prefer a more explicit check
```suggestion
  if (selectedRoutes.length === 0) {
```

Reply:
i updated this